### PR TITLE
Use direct WS connection instead of proxied one

### PR DESCRIPTION
**Pull request by karelhala**
### Description

Akamai's WS proxy is not working correctly, let's use direct WS url instead.

### CODE REVIEW

File: .codecov.yml [hunk 1]
```diff
@@ -3,10 +3,5 @@ coverage:
...
   range: "0...100"
   status:
-    project:
-      default:
-        if_ci_failed: success
-    patch:
-      default:
-        if_ci_failed: success
-
+    project: false
```
Comment (florkbr) [inline .codecov.yml:6]: @karelhala I don't think we want to undo this change in this PR?

Comment (karelhala) [reply .codecov.yml:6]: @florkbr the runs were still breaking the CI. This disables the runs altogether, will post the comment about status, but won't run the jobs. I got fed up with the config and added it here. I could create a new PR if you want to.

**Comment by florkbr** [comment]
Approving to unblock development. We will need to revert when web sockets are fixed through Akamai
